docs/docs/animations/withTiming.mdx

```diff
@@ -54,6 +54,26 @@ enum ReduceMotion {
 #### `toValue`
 
 The value on which the animation will come at rest.
+There are many possible types for `toValue`, currently the following kinds of values are supported:
+
+- **colors** - we support wide range of color formats:
+  - hex numbers - `0xff1234`
+  - rgb - `rgb(100,50,0)`
+  - rgba - `rgba(255,105,180,0)`
+  - hsl - `hsl(0, 50%, 50%)`
+  - hex strings - `#53575E`
+  - css color names - `dodgerblue`
+- **numbers**
+- **suffixed numbers** - strings being a number with a unit, like `"100%"` or `"90deg"`
+- **objects** - object with properties that will be animated separately
+- **array** - array of values, each value will be animated separately.
```

Comment:
```suggestion
- **array** - array of numbers, each value will be animated separately,
```
any values? i think it only works with numbers

Reply:
Yes, it does, I've just verified